tests/layer_tests/conftest.py

```diff
@@ -65,10 +66,10 @@ def pytest_addoption(parser):
         action="store_true",
         help="Use Model Optimizer with new FrontEnd")
     parser.addoption(
-        "--api_2",
-        action="store_true",
-        help="Use new API 2.0 for model processing in Inference Engine",
-        default=True)
+        "--use_old_api",
```

Comment:
will we have a key to test with the new frontend? Now it is the legacy one by default.
Also, to use simultaneously old API and new TF FE frontend is incompatible.

Reply:
Yes, we already have --use_new_frontend key. I added configuration for CI job in nightly with new API and new FE usage

Reply:
@rkazants no, we do not have such check, but purpose for this will be only for local runs bcz we have no such runs in CI.

Reply:
Please add an explicit comment about this restriction in the help usage or please add the check. It helps to avoid the confusion among developers who will use the layer tests for TF FE testing.

Reply:
I added check for new FE and old API usage in openvino/tests/layer_tests/conftest.py file, other changes are ctrlF + ctrlR changes